packages/repack/commands.js

```diff
@@ -50,6 +50,117 @@ function getCommands() {
   return commands;
 }
 
+const startCommandOptions = [
+  {
+    name: '--cert <path>',
+    description: 'Path to custom SSL cert',
+  },
+  {
+    name: '--host <string>',
+    description: '(default: "")',
+  },
+  {
+    name: '--http',
+    description: 'Enables https connections to the server',
+  },
+  {
+    name: '--key <path>',
+    description: 'Path to custom SSL key',
+  },
+  {
+    name: '--port <number>',
+    description: 'The port number that runs the server on',
+  },
+  {
+    name: '--no-interactive',
+    description: 'Disables interactive mode',
+  },
+  {
+    name: '--silent',
+    description: 'Silents all logs to the console/stdout',
```

Comment:
is this working as expected?

Reply:
yup, no logs from server with this switch
![image](https://github.com/callstack/repack/assets/7682108/460d9df7-e037-4547-9054-fb6f75700f5b)